fvm/fvm_test.go

```diff
@@ -525,7 +525,7 @@ func TestBlockContext_DeployContract(t *testing.T) {
 		err = vm.Run(ctx, tx, ledger, programs.NewEmptyPrograms())
 		require.NoError(t, err)
 
-		assert.NoError(t, tx.Err)
+		assert.ErrorContains(t, tx.Err, "program too ambiguous")
```

Comment:
This contract deployment now fails. I have changed the test to expect a "program too ambiguous" error. Is this ok/expected @turbolent.

Reply:
I used the solution from here https://axiomzen.slack.com/archives/CG0B7CJAJ/p1653502279521059